src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -74,14 +75,15 @@ def test_analyze_list(self, multihost, backupsssdconf):
         multihost.client[0].run_command(i_cmd, raiseonerr=False)
         for act_op in ['list', 'list -v']:
             _, stdout = analyze(multihost, act_op)
-            assert all(ptn in stdout for ptn in ['id', 'getent'])
+            assert all(ptn in stdout for ptn in [' id', 'getent'])
```

Comment:
why this change for the old multihost test?

Reply:
Good question.
Look at the format of the output of analyze command:
```
2023-12-04  8:27:49: [uid 0] CID #1: id
2023-12-04  8:27:49: [uid 0] CID #2: sshd: user1
2023-12-04  8:27:49: [uid 0] CID #3: systemd-userwork
2023-12-04  8:27:49: [uid 0] CID #4: systemd-userwork
```
There is always [uid X] which would match our searched string 'id' so I added additional backspace.